src/ctia/task/migration/store.clj

```diff
@@ -625,11 +617,10 @@ when confirm? is true, it stores this state and creates the target indices."
   [stores :- {s/Keyword MigratedStore}
    prefix :- s/Str
    services :- MigrationStoreServices]
-  (->> (map (fn [[store-key raw]]
-              {store-key (-> (with-store-map store-key prefix raw services)
-                             update-source-size)})
-            stores)
-       (into {})))
+  (into {} (map (fn [[store-key raw]]
+                  {store-key (-> (with-store-map store-key prefix raw services)
+                                 update-source-size)}))
+        stores))
```

Comment:
Is this also about the transducer driving a different code path? I'm curious about the why of this.